File: JSBASIC/126_power_loop.js
// 126	126_power_loop.js	Calculate the power of a number using a loop (without ** operator).

let base = 3, exponent = 4;
let result = 1;
for (let i = 1; i <= exponent; i++) {
    result *= base;
}
console.log("Base:", base, "| Exponent:", exponent);
console.log("Result:", result);

File: JavaBasic/126_power_loop.java
// 126	126_power_loop.java	Calculate the power of a number using a loop (without ** operator).
public class _126_power_loop {
    public static void main(String[] args) {
        int base = 3, exponent = 4, result = 1;
        for (int i = 1; i <= exponent; i++) result *= base;
        System.out.println(base + "^" + exponent + " = " + result);
    }
}
